packages/conform-dom/index.ts

```diff
@@ -34,10 +31,7 @@ export {
 } from './submission';
 export {
 	getFormData,
-	getPaths,
-	formatPaths,
+	getPathSegments as getPaths,
+	formatPathSegments as formatPaths,
```

Comment:
I have renamed several helpers internally. But still exported in the old names with the exact same signature. So it shouldn't breaking any people.